metadata-ingestion/src/datahub/ingestion/source/looker/lookml_sql_parser.py

```diff
@@ -214,20 +260,51 @@ def create_or_update_fields(
         if `self.fields` is None that means view is defined as per Syntax2.
         """
 
-        query: str = sql_query.sql_query()
+        query: str = self.sql_query.sql_query()
+
+        logger.debug(f"Processing query: {query} for view-urn {view_urn}")
 
         spr: SqlParsingResult = create_lineage_sql_parsed_result(
             query=query,
             default_schema=connection.default_schema,
             default_db=connection.default_db,
             platform=connection.platform,
             platform_instance=connection.platform_instance,
-            env=env,
-            graph=ctx.graph,
+            env=cast(str, connection.platform_env),  # It's never going to be None
```

Comment:
if this is true, let's just change the type of connection.platform_env to not be optional

Reply:
It will break existing recipe, please check this [L108](https://github.com/sid-acryl/datahub-fork/blob/master%2Bing-510-lookml-cll/metadata-ingestion/src/datahub/ingestion/source/looker/lookml_config.py#L108) and why it is never going to be None, see below comment

Reply:
why don't we use `env=connection_def.platform_env or self.source_config.env,`, like we do in other places currently?

Reply:
Done